### DISCUSSION TITLE
New to 86Box

### DISCUSSION THREAD

**Discussion by EPICNESS2008** [86Box/86Box#4878] [Q&A] [answered]
Ok, so, I just recently got 86Box and I can't find the roms directory. I looked in my Local folder in my AppData, I looked in the Program Files and I can't find it. Do I need to make a roms directory in the extracted folder?

**Comment by parkerlreed** [comment, marked as answer]
Yes you can have the `roms` folder in the same folder as the binary.

The docs go over this fairly well https://86box.readthedocs.io/en/latest/usage/roms.html

**Comment by Pellzi** [comment]
Is it possible, that the Roms, 86Box.exe use, will be directly compiled into the 86Box.exe?
Even when the program will be approximative 200% greater than wow?

This prevents such Threads, where are the Roms. And further, then the Roms are there, where they should be be. 
I you really want to debug and patch Roms for the 86Box, the new 86Box with included Roms should give the user a opportunity to export and import the patched Roms inside the 86Box.exe. The new 86Box with this feature works then like the Dos Programm Setver.exe .
Which change the CRC32 Control String in dependency of what is inside written of setver.exe .

**Reply by seal331** [reply]
Some emulators compile ROMs into their binaries and have been doing so for a long time now, but the issue here (at least to me) looks like legal. All the various ROMs used by 86box (except a select _very_ few) are still under copyright by whoever developed them back in the 80s/90s/00s, and while most of those companies don't exist these days or don't care about people redistributing 30 year old software (well, to be frank, 98% of the entire old software scene relies on these companies or their successors not caring about people redistributing ancient code they still technically own), that still brings countless issues about distribution. I've seen plenty of Linux distros which don't package programs with questionable/proprietary licenses, or at most put them in some use-discouraged non-free repo, separate from the main repo. Not to mention people with bad internet connection who may want to run some 1981 PC game which only works on an original 5150 having to download 88 megabytes of stuff they won't ever use...

Please note that I'm not a lawyer though, and this is not legal advice, this is just my personal opinion.

**Reply by TC1995** [reply]
Just a big no.

**Reply by Pellzi** [reply]
The Copyright problem, i have fully forgotten, during this Idea.
And i thought, the 86Box will be probably supported by Hardware Manufactures with original Biose, and documentations, which are not freely available. But it seems to be, that is not the case.

Has the 86Box Team ever tried to contact these Manufacturers? With the wish, to become a official License from them, for nothing, for the used Biose? Probably there is a Chance,  if someone of the Team made this try?
Because the 86Box Project is the only One, (I know), which wants to Emulate such old Machines in just one Emulator. With the Wish, to have as many Bioses before the Year 2000 as is possible.
And this with the Fortune for the Users, that OBattler does not stop to Debug, without blaming any One, who made this programming fault in the Past.  And the 86Box Team knows probably such old System better than every body else on this planet. 

Probably, this reason is the definite OK for these Manufacturers to give 25 or more Years later a official License to use this Bios to the 86Box Team for the official Use in the 86Box Project.
